components/weather_data/WeatherStationsDetail.tsx

```diff
@@ -164,7 +269,7 @@ const TimeSeriesTable: React.FC<{timeSeries: WeatherStationTimeseries}> = ({time
               name={shortFieldMap[field]}
               units={shortUnits(timeSeries.UNITS[field])}
               elevation={elevation ? elevation.toString() : ''}
-              data={times.map(time => (time in dataByTime ? (dataByTime[time] === null ? '-' : String(dataByTime[time])) : '-'))}
+              data={times.map(time => (dataByTime[time] !== null ? String(dataByTime[time]) : '-'))}
```

Comment:
@stevekuznetsov in the process of rebasing on #594 it seemed to me that this could be simplified quite a bit - double ternary is hard to parse. Give a shout if this looks wrong to you!